src/components/Navbar/index.tsx

```diff
@@ -56,16 +59,30 @@ export class NavbarBase extends React.Component<Props> {
   }
 
   render() {
-    const { user, currentVersion } = this.props;
+    const { currentVersion, reviewersHost, user } = this.props;
 
     return (
       <Navbar className={styles.Navbar} expand="lg" variant="dark">
         <Navbar.Brand className={styles.brand}>
           <div className={styles.info}>
             {currentVersion && (
-              <div className={styles.infoItem}>
-                {getLocalizedString(currentVersion.addon.name)}
-              </div>
+              <>
+                <div
+                  className={makeClassName(styles.infoItem, styles.addonName)}
+                >
+                  {getLocalizedString(currentVersion.addon.name)}
+                </div>
+                <div className={styles.infoItem}>
+                  <a
+                    className={styles.reviewerToolsLink}
```

Comment:
The blue link is hard to read against the dark gray. I suggest making the text white and underlining it to make it clear that it's a link.